### ISSUE TITLE
warning when install via homebrew

### ISSUE THREAD

**Issue by zhuangya** [beyondgrep/ack1#217]
i installed ack via homebrew just now, and there is a warning there:

```
 $ brew install ack
==> Downloading https://github.com/petdance/ack/tarball/1.96
######################################################################## 100.0%
==> pod2man ack ack.1
Warning: inreplace in 'ack.1' failed
Expected replacement of '\*(d\-' with '\*(d-'
/usr/local/Cellar/ack/1.96: 6 files, 168K, built in 28 seconds
```

**Comment by petdance** [comment]
I'm sorry to hear about this.  I've never used homebrew, so I can't address the problem.  I have nothing to do with what homebrew does or how it does it.  I'm especially disappointed that it's pulling from a tarball from GitHub instead of from the CPAN where it should be pulling from.

If there's something that I can help with, I will.  However, whoever put together the homebrew package (Is that even how it works?) will have to fix this problem.

**Comment by zhuangya** [comment]
well, homebrew is just a package manger for Mac OS X.

after some digging, i found that it's the problem here:

  https://github.com/mxcl/homebrew/blob/master/Library/Formula/ack.rb 

so, do you have any idea about that `pod2man formatting error`

and i'll report this to homebrew :)

**Comment by MikeMcQuaid** [comment]
Sorry this issue was reported here @petdance and it's been fixed. Thanks for the great tool.

We would pull from CPAN if that's a better place for the standalone version but it currently doesn't supply the ZSH completion that the GitHub repository does.

**Comment by zhuangya** [comment]
it's my fault that do not figure out where the problem is.

great love `ack`. :)

**Comment by petdance** [comment]
I guess it doesn't much matter if you get the tarball from GitHub or CPAN, except that the CPAN is the official repository.  If I ever moved the repository from GitHub, then the brew recipe would break.

This DOES identify that the .zsh completion never made it into the release tarball.  I've fixed that with 4d705b59658f47faab8905ea440de44a95861105, so good has come from this.  Thanks!
